slave.mk

```diff
@@ -336,7 +336,7 @@ $(SONIC_INSTALL_WHEELS) : $(PYTHON_WHEELS_PATH)/%-install : .platform $$(addsuff
 
 # start docker daemon
```

Comment:
Suggest updating comment to reflect new functionality: `start docker daemon if not already started`

Reply:
changed, thanks.

Reply:
Doesn't appear to have changed, FYI. Change not staged?